Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(grunt) {
     cssPath: "webcompat/static/css",
     imgPath: "webcompat/static/img",
     banner: "/*! <%= pkg.title %>\n" +
-      " *  Copyright (c) <%= grunt.template.today(\"yyyy\") %> <%= pkg.author.name %>\n" +
+      ' *  Copyright (c) <%= grunt.template.today("yyyy") %> <%= pkg.author.name %>\n' +
```

Comment:
Seems a little funky to use a single quote here, when the lines above and below use double quotes. Any reason why?